sup/include/sup/ioutil.hpp

```diff
@@ -32,7 +32,7 @@ class basic_null_streambuf: public std::basic_streambuf<charT, traitsT> {
     typedef typename streambuf_type::off_type off_type;
     typedef typename streambuf_type::traits_type traits_type;
 
-    virtual ~basic_null_streambuf() {}
+    virtual ~basic_null_streambuf() = default;
```

Comment:
Also not the same behavior.

Reply:
As far as I can see that's not the case. The default deconstruction via `= default` will destroy all members (in reverse order), then the base classes. The same is true of `{}`. So, no difference, right?